### PR TITLE
Add SEO cheatsheet

### PR DESCRIPTION
**Pull request by Hari70a**
## What does this PR do?
Add SEO cheatsheet

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Hari70a** [comment]
closes https://github.com/EbookFoundation/free-programming-books/issues/7681

**Comment by Hari70a** [comment]
@davorpa , @eshellman

Kindly review it.

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -136,6 +136,7 @@
 * [HTML & CSS Emmet Cheat Sheet](https://docs.emmet.io/cheat-sheet/) - Emmet Documentation (HTML, [PDF]( https://docs.emmet.io/cheatsheet-a5.pdf))
 * [HTML CheatSheet](https://htmlcheatsheet.com) - htmlcheatsheet.com (HTML, [PDF](https://htmlcheatsheet.com/HTML-Cheat-Sheet.pdf))
 * [SCSS CheatSheet](https://devhints.io/sass) - devhints, Rico Santa Cruz (HTML)
+* [SEO CheatSheet](https://learn-the-web.algonquindesign.ca/topics/seo-cheat-sheet/) - Learn the Web, Algonquin Design (HTML)
```
Comment (eshellman) [inline more/free-programming-cheatsheets.md:139]:
```suggestion
* [SEO CheatSheet](https://learn-the-web.algonquindesign.ca/topics/seo-cheat-sheet/) - Algonquin Design (HTML)
```

Comment (Hari70a) [reply more/free-programming-cheatsheets.md]: https://github.com/EbookFoundation/free-programming-books/pull/7752/commits/7d7bf1221e580d0cf590df1ca363226bb357c81d

Done @eshellman

**Pull request merged**